Lib/sqlite3/_completer.py

```diff
@@ -5,15 +5,20 @@
 except ImportError:
     SQLITE_KEYWORDS = ()
 
+SQLITE_KEYWORDS += ('.quit', '.help', '.version')
```

Comment:
Sorry for the delay, the notification got buried.

These are not SQLite keywords. SQLite keywords are case-insensitive and don't start with a dot.
Could you use a different variable, like `CLI_DOT_COMMANDS`?

Also, could you add a comment to `__main__.py` reminding people to update this if they change the commands? (That's the easy option; a follow-up PR could move logic from `__main__.py` to `cli.py` so the command list could be imported from `cli.py`, and kept close to the implementation.)